### DISCUSSION TITLE
Geomap distance measurement tools

### DISCUSSION THREAD

**Discussion by HenrikRolandHansen** [grafana/grafana#43126] [Dashboards, Panels & Visualizations]
I would be nice with the possibility to measure distance between to or serval points within the GeoMap map.

**Comment by nmarrs** [comment]
Hi @HenrikRolandHansen, good news! With Grafana 9.1 we introduced measure tools to geomap. See the [documentation here](https://grafana.com/docs/grafana/next/panels-visualizations/visualizations/geomap/#show-measure-tools)

**Reply by HenrikRolandHansen** [reply]
Thanks for info @nmarrs already using it. And it is just grate👍

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
